token/cli/src/main.rs

```diff
@@ -2702,6 +2803,23 @@ async fn process_command<'a>(
                 config.pubkey_or_default(arg_matches, "mint_authority", &mut wallet_manager);
             let memo = value_t!(arg_matches, "memo", String).ok();
 
+            let mut rate_authority: Option<Pubkey> = None;
+            let mut rate_bps: Option<i16> = None;
+            let interest_bearing =
+                values_t!(arg_matches.values_of("interest_bearing"), String).ok();
+            if let Some(values) = interest_bearing {
+                let rate_addr = &values[0];
+                if rate_addr != "NONE" {
+                    is_valid_pubkey(rate_addr)?;
+                    rate_authority = Some(config.pubkey_or_default(
+                        arg_matches,
+                        "interest_bearing",
+                        &mut wallet_manager,
+                    ));
+                }
+                rate_bps = Some(values[1].parse::<i16>().unwrap());
+            }
+
```

Comment:
this becomes a much more pleasant `values_t!` call if we change the flag to one param